hw2/04-background.js

```diff
@@ -1 +1,49 @@
 // Add your code here
+const button = document.getElementById("button");
+const time = document.getElementById("time");
+let setTimeInterval;
+let colorChange = true;
+let lbl = document.getElementById("myLabel");
+let warningLabel = false;
+const randomRGB = function generateRandomNumber() {
+  return Math.floor(Math.random() * 255);
+};
+
+const generateColor = function generateColor() {
+  document.body.style.background =
+    "rgba(" + randomRGB() + "," + randomRGB() + "," + randomRGB() + ",0.3)";
+};
+
+const changeColor = function changeColor() {
+  if (time.value > 0 && time.value !== null) {
+    lbl.innerHTML = "Background change interval (seconds):";
+    lbl.style.color = "black";
+    warningLabel = false;
+    button.innerHTML = "Stop";
+    button.className = "btn btn-danger";
+    setTimeInterval = setInterval(generateColor, time.value * 1000);
+  } else {
+    if (!warningLabel) {
```

Comment:
Small style suggestion: you can save one set of curly brackets by combining the else and if. 
```suggestion
  } else if (!warningLabel) {
```

Reply:
fixed